deploy-commands-test.js

```diff
@@ -1,45 +1,45 @@
-const { REST } = require('@discordjs/rest');
-const { Routes } = require('discord-api-types/v9');
-const fs = require('fs');
+const { REST } = require("@discordjs/rest");
+const { Routes } = require("discord-api-types/v9");
+const fs = require("fs");
 
-const dotenv = require('dotenv');
+const dotenv = require("dotenv");
 dotenv.config();
 
 // XPHolder TEST bot user ID
-const CLIENT_ID = "1187964800114901093";
+const CLIENT_ID = process.env.CLIENT_ID;
 // Enter Ravenloft TEST server ID
-const TESTING_SERVER_ID = "1068243670391857192";
+const TESTING_SERVER_ID = process.env.CLIENT_ID;
```

Comment:
I think you mean `process.env.TESTING_SERVER_ID`, right?

Actually, do we maybe want a new environment variable `PROD_SERVER_ID`? The reason I split it into "deploy-commands-prod" and "deploy-commands-test" was to make sure I was deploying the changes on the correct server!

Reply:
So I had it as a singular `CLIENT_ID` so that we can use workflows and automate deployment once something is merged in. The client id would change based on what environment you're deploying to and depend on the env settings. That way we don't ever actually set up production on our local machines and instead only have production values set up on the production environment, thus allowing us to more closely mirror production on our local because we are using the same script to handle the deploy.